manifests/resource-topology-exporter-ds.yaml

```diff
@@ -29,7 +29,7 @@ roleRef:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: resource-topology-exporter-ds
+  name: rte
```

Comment:
I think it is better to leave it as `resource-topology-exporter-ds` here because we have been using `rte` as namespace in a lot of places. Even though I know it wouldn't cause any confusion but I don't want it to appear to the user might thing that the daemon set name has something to do with the watch-namespace.

Reply:
sure, fixed